pact-repl/Main.hs

```diff
@@ -76,6 +79,11 @@ loadFlag = fmap OLoad $
     <*> O.argument O.str
       (O.metavar "FILE" <> O.help "File path to compile (if .pact extension) or execute.")
 
+checkNativeShadowingFlag :: O.Parser ReplOpts
```

Comment:
Shouldn't this be a flag (no filename) ?

Reply:
This one isn't a flag for the repl, this allows you to check shadowing on a file without actually running it. 

In the repl, checking native shadowing will be enabled unconditionally.